nmostesting/suites/IS0502Test.py

```diff
@@ -593,3 +594,206 @@ def test_13(self, test):
+                                                    component["height"] != (flow["frame_height"] / 2):
+                                                sampling_error = True
+                                        if sampling_error:
+                                            return test.FAIL("Video Flow {} components do not match the expected "
+                                                             "dimensions for Sender sampling {}"
+                                                             .format(flow["id"], sampling))
+                            elif param_components[0] == "width":
+                                if flow["frame_width"] != int(param_components[1]):
+                                    return test.FAIL("Width for Sender {} does not match its Flow {}"
+                                                     .format(resource["id"], flow["id"]))
+                            elif param_components[0] == "height":
+                                if flow["frame_height"] != int(param_components[1]):
+                                    return test.FAIL("Height for Sender {} does not match its Flow {}"
+                                                     .format(resource["id"], flow["id"]))
+                            elif param_components[0] == "depth":
+                                for component in flow["components"]:
+                                    if component["bit_depth"] != int(param_components[1]):
+                                        return test.FAIL("Bit depth for Sender {} does not match its Flow {}"
+                                                         .format(resource["id"], flow["id"]))
+                            elif param_components[0] == "colorimetry":
+                                if flow["colorspace"] != param_components[1].split("-")[0].replace(".", ""):
+                                    return test.FAIL("Colorimetry for Sender {} does not match its Flow {}"
+                                                     .format(resource["id"], flow["id"]))
+                            elif param_components[0] == "interlace":
+                                if "interlace_mode" not in flow or flow["interlace_mode"] == "progressive":
+                                    return test.FAIL("Interlace parameter for Sender {} does not match its Flow {}"
+                                                     .format(resource["id"], flow["id"]))
+                            elif param_components[0] == "top-field-first":
+                                if "interlace_mode" not in flow or flow["interlace_mode"] != "interlaced_tff":
+                                    return test.FAIL("Top-field-first parameter for Sender {} does not match its Flow "
+                                                     "{}".format(resource["id"], flow["id"]))
+                            elif param_components[0] == "segmented":
```

Comment:
Should this also check `len(param_components) == 1`?

Reply:
Yes